ea_server/data/ea_request_model.py

```diff
@@ -0,0 +1,56 @@
+from dataclasses import dataclass
+from typing import List, Dict
+
+from dacite import MissingValueError
+
+from ea_server.api.utils.constans import INIT_ORDER_ID
+
+
+@dataclass(frozen=True)
+class Driver:
+    id: str
+    max_capacity: int
+    max_distance: int
+
+
+@dataclass(frozen=True)
+class Order:
+    id: str
+    weight: int
+
+
+@dataclass(frozen=True)
+class Data:
```

Comment:
Maybe we can think about a more descriptive name?

Reply:
I chose to call it like that because I looked at it from the request point of view, 
but I can change it to EaData if u think it more descriptive